src/main/scala/com/fulcrumgenomics/sopt/Sopt.scala

```diff
@@ -79,8 +102,17 @@ object Sopt {
                  maxValues: Int,
                  defaultValues: Seq[String],
                  sensitive: Boolean,
-                 description: String
-                ) extends MarkDownDescription
+                 description: String,
+                 isSpecial: Boolean,
+                 userValue: Option[Seq[String]]
```

Comment:
I don't like this.  These classes are supposed to be metadata about the class/args.  Stuffing values into them is weird and makes then seem more like the internal implementation classes.

Reply:
1. I don't think `userValue` is correct, as it could also be the default value.
2. The use case I have is inspecting the class built from command line parsing, so I want to know the value.  For example, I have built a tool from the arguments on the command line, now I want to know for a given argument for that tool, does it have a value and if so, what is it?